web/src/Sidebar.test.tsx

```diff
@@ -246,4 +246,31 @@ describe("sidebar", () => {
 
     expect(tree).toMatchSnapshot()
   })
+
+  it("disables trigger button for Tiltfile", () => {
```

Comment:
can you elaborate on the value of this test? you don't seem to be asserting anything

Reply:
I mean I'm matching against a snapshot in which the trigger button is `disabled` and has the appropriate tooltip. Is your question about what I am/am not expecting to be in the snapshot, or about the ethos of snapshot tests more broadly?

Reply:
this test is a pretty good model:
https://github.com/windmilleng/tilt/blob/master/web/src/SidebarTriggerButton.test.tsx#L38